test/unittests/test_huffman.cpp

```diff
@@ -38,40 +62,40 @@ TEST_F(TestHuff, HUFF1) {
     std::vector<std::uint64_t> encInput{0b01U, 0b10U};
 
     for (const auto& in1: encInput) {
-        auto search = tt.findCubeInteger(in1, 2U);
-        EXPECT_TRUE(search != tt.cubeMap.end());
+        auto search = tt.find(in1, 2U);
+        EXPECT_TRUE(search != tt.end());
         EXPECT_TRUE(search->second.equals(std::string("0-")));
```

Comment:
```suggestion
        EXPECT_TRUE(search->second.equals("0-"));
```
the cast to `std::string` is unnecessary here. please also check in all other cases/files.

Reply:
done 👍🏼